src/components/modals/home/StepTwoRegistration.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from "react";
+import Button from "../../commons/Button";
+import Input from "../../commons/Input";
+import { IRegistrationModalStepTwo } from "../../../types/types";
+import Link from "next/link";
+import { Checkbox } from "antd";
+import closeModalHandler from "@/actions/closeModalHandler";
+import { message } from "antd";
+import { Modal } from "antd";
+
+const StepTwoRegistration: React.FC<IRegistrationModalStepTwo> = ({
+  content,
+  label,
+  isOpen,
+  onClose,
+  secretKey,
+  publicKey,
+}) => {
+  const [isChecked, setIsChecked] = useState<boolean>(false);
+
+  const handleCancelChecked = () => {
```

Comment:
Same here

Reply:
solved in commit 424537b8225447b094604e1bc6f94afd41adf270.